app/helpers/footer_helper.rb

```diff
@@ -1,11 +1,31 @@
 module FooterHelper
   def footer_meta_items
+    if current_service == :claims
+      claims_footer_meta_items
+    else
+      placements_footer_meta_items
+    end
```

Comment:
Let's avoid using an `else` statement. If `current_service` is not either `:claims` or `:placements`, we want to immediately be able to see that.

```suggestion
    {
      claims: claims_footer_meta_items,
      placements: placements_footer_meta_items
    }.fetch current_service
```